### PR TITLE
Commandline tool

### PR DESCRIPTION
**Pull request by etopeter**
Proposal to add commandline tool
create new password:
encpass.sh -s my_key.enc

To get password
encpass.sh -g my_key.enc

use in shell scripts
. encpass.sh -q

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
 ._*
+*.pem
```
Comment (ahnick) [inline .gitignore]: Is this for when running the test.sh script?

**Comment by ahnick** [comment]
I had been considering this already.  Let me review these changes for POSIX compliance.  Ideally I'd like to support this kind of use case, but I'd still like this to work in a wide array of different shells.  (csh, zsh, tcsh, etc.)  I probably need some kind of test harness for the POSIX compliance.  If you have recommendations let me know.

File: .gitignore (hunk 1 above)
Comment (etopeter) [reply .gitignore]: Yes test script and any other runs you would do in context of current directory.

**Comment by etopeter** [comment]
I know this PR is a bit of a rewrite and changes dynamics of the tool. My aim was to have both functionalities as far as sourcing it in scripts and using functions , and running as a command line tool to create new secrets. This code was tested only on mac. I used getopts POSIX compliant short parameters.

**Pull request closed without merging**